addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/welcomePerson.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="netatmo"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+      
+    <thing-type id="NAWelcomePerson">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="netatmoapi" />
+        </supported-bridge-type-refs>
+        
+        <label>Welcome Camera Person</label>
+        <description>This represents a person at home</description>
+
+        <channels>
+            <channel id="welcomePersonId" typeId="id"></channel>
+        	<channel id="welcomePersonLastSeen" typeId="last_seen"></channel>
+        	<channel id="welcomePersonOutOfSight" typeId="out_of_sight"></channel>
+        	<channel id="welcomePersonFaceID" typeId="face_id"></channel>
+        	<channel id="welcomePersonFaceVersion" typeId="face_version"></channel>
+            <channel id="welcomePersonFaceKey" typeId="face_key"></channel>
+            <channel id="welcomePersonPseudo" typeId="pseudo"></channel>
+            <channel id="welcomePersonAtHome" typeId="person_athome"></channel>
+            <channel id="welcomePersonLastEventID" typeId="person_eventid"></channel>
+            <channel id="welcomePersonLastEventMessage" typeId="person_eventmsg"></channel>
+            <channel id="welcomePersonLastEventTime" typeId="person_eventtime"></channel>
+            
+            <channel id="welcomePersonAvatarPictureUrl" typeId="person_avatar_picture_url"></channel>
+            <channel id="welcomePersonLastEventPictureUrl" typeId="person_event_picture_url"></channel>                        
+        </channels>
+            
+        <properties>
+             <property name="actualApp">app_welcomecamera</property>
+        </properties>         
+        
+        <config-description>
+            <parameter name="welcomeHomeId" type="text" required="true" readOnly="true">
+                <label>Home ID</label>
+                <description>Id of the home</description>
+             </parameter>
+
+            <parameter name="welcomePersonId" type="text" required="true" readOnly="true">
```

Comment:
I get the welcomePersonId from the netatmo api, don't know if its a good idea to make it possible to manually change it, whats the preferred implementation?

For some api calls, the welcomePersonId is used so i get it back as channel to use it in scripts or rules, don't know if you can get this information there also from the parameters